### PR TITLE
Feature/collecting disks

### PR DESCRIPTION
**Pull request by Big3ppvv**
### Collecting Disks
**Make a new exercise of collecting objects like disks**

### CODE REVIEW

**Review by Big3ppvv — COMMENTED** [review]
I added a new exercise

**Review by Arsentau — CHANGES_REQUESTED** [review]
Why did you delete the previous exercices?

File: ej4-collectingDisks/exercise4.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+function creatingDisks(id, propiedad, discos, valor) {
+    if(valor === ""){
```
Comment (Arsentau) [inline ej4-collectingDisks/exercise4.js:2]: you are mixing Spanish and English, please fix it

File: ej4-collectingDisks/exercise4.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        delete discos[id][propiedad];
+
+    }else if(propiedad === "canciones" && discos[id][propiedad] === undefined) {
```
Comment (Arsentau) [inline ej4-collectingDisks/exercise4.js:5]: this else if concatenation is making it hard to read, is there another way to implement this?

File: ej4-collectingDisks/exercise4.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+    }else if(propiedad === "canciones" && valor !== ""){
+        discos[id][propiedad].push(valor);
+
+    }else if(propiedad !== "canciones" && valor !== "")
+        discos[id][propiedad] = valor;
+
+}
+
+var coleccionDeDiscos = {
```
Comment (Arsentau) [inline ej4-collectingDisks/exercise4.js:16]: `var` is forbbiden

File: ej4-collectingDisks/exercise4.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        canciones : ["Stayin Alive"]
+    },
+
+    5439 : {
+        titulosDelAlbum : "ABBA Gold"
+    }
+};
+
+const id = 4589;
+var propiedad = "canciones";
```
Comment (Arsentau) [inline ej4-collectingDisks/exercise4.js:29]: please not make usage of `var` to declare a variable

**Review by Arsentau — CHANGES_REQUESTED** [review]

File: README.md
[the remark this answers is not in the document]
Comment (Big3ppvv) [reply README.md:1]: I don't deleted them. Only was this 2 exercises. Do i have to put n3 and n4?

**Comment by Arsentau** [comment]
It seems you have conficts

File: README.md [hunk 5]
```diff
@@ -4,3 +4,5 @@
 
 - [Ejercicio-1](./ej1-return_element/README.md)
 - [Ejercicio-2](./ej2-golf_message/README.md)
+- [Ejercicio-3](./ej3-couting_Cards/README.md)
```
Comment (Arsentau) [inline README.md:7]: link to exercice 4 is missing

**Review by Arsentau — CHANGES_REQUESTED** [review]
The exercice 4 doesn't have its own readme file
Every file MUST have an empty new line tailing space
The name of functions must be fixed, it should be camel case, not snake case

**Pull request merged**